build.gradle

```diff
@@ -190,6 +209,34 @@ processResources {
    from 'LICENSE.txt'
 }
 
+// License header formatting
+license {
+    ext {
+        name = "Mixin"
+        organization = "SpongePowered"
+        url = "https://www.spongepowered.org"
```

Comment:
Should we tweak this?

Reply:
No, as this will then require us to change all of the licnese headers.